packages/aws-iam-authenticator/aws-iam-authenticator.spec

```diff
@@ -14,6 +14,7 @@ Summary: AWS IAM authenticator
 License: Apache-2.0
 URL: https://%{goimport}
 Source0: https://%{goimport}/archive/v%{gover}/%{gorepo}-%{gover}.tar.gz
```

Comment:
Should this be referencing the upstream path? In our other packages, as far as I understand it, the bundle gets downloaded from its reference in `Cargo.toml`, then the `Source0` line typically just refers to this local `tar.gz` file.

https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/hotdog/hotdog.spec#L16

Reply:
Hmm, I guess this is pretty normal. Looking through the other packages, most of them actually do refer to it this way rather than the local file name.